setup.py

```diff
@@ -45,7 +45,7 @@
         "transformers>=4.34.0",
     ],
     "openvino": ["openvino>=2023.2", "onnx", "onnxruntime", "transformers>=4.36.0", "optimum>=1.16.1"],
-    "nncf": ["nncf>=2.7.0"],
+    "nncf": ["nncf @ git+https://github.com/openvinotoolkit/nncf.git"],
```

Comment:
for when is the next nncf release planned ?

Reply:
The release should be in 2-3 weeks. I would like to migrate to develop version for now to have time for integration.

Reply:
OK would prefer to wait for nncf next release but 2/3 weeks sounds reasonable